flink-connector-postgres-cdc/src/main/java/com/ververica/cdc/connectors/postgres/source/fetch/PostgresStreamFetchTask.java

```diff
@@ -128,6 +128,18 @@ public void commitCurrentOffset() {
                             postgresOffsetContext
                                     .getOffset()
                                     .get(PostgresOffsetContext.LAST_COMMIT_LSN_KEY);
```

Comment:
It no longer need this code, because only streamingSlit is allowed to commitCurrentOffset. More over, `LAST_COMMIT_LSN_KEY` is update by `Begin` or `Commit` rather than message (See PostgresStreamingChangeEventSource#processMessages -> PostgresStreamingChangeEventSource#commitMessage -> PostgresOffsetContext#updateCommitPosition).

Reply:
> It no longer need this code, because only streamingSlit is allowed to commitCurrentOffset. More over, `LAST_COMMIT_LSN_KEY` is update by `Begin` or `Commit` rather than message (See PostgresStreamingChangeEventSource#processMessages -> PostgresStreamingChangeEventSource#commitMessage -> PostgresOffsetContext#updateCommitPosition).

Thanks for your review, I will modify the code according to your suggestion。
 Yes, your are right, LAST_COMMIT_LSN_KEY is updated by Begin or commit message. 

I want to explain the data lost problem in more detail, because the problem happens on our online when failover. After apply this pr's fix,  there have been no data loss so far.

If checkpoint succeeds and before notifyCheckpoint's commit confirmed_flush_lsn finishes, there's a table UPDATE event arrives, the event appears as BEGIN/UPDATE/COMMIT LSN sequence, the notifyCheckpoint will commit COMMIT's LSN to slot. If failover happends at this time and will restore from this checkpoint, it will begin consume from the slot's confirmed_flush_lsn(that is COMMIT's LSN), nor the LSN of the streamsplit's lsn of checkpoint, because streamsplit's lsn < slot's confirmed_flush_lsn。From the high level view, the table's UPDATE event is lost.

Reply:
Thanks a lot for your explain, I totally agree with you.  Moreover, I have also found that Postgres offset commit is no bound to checkpoint id before your PR.  If do checkpoint five consecutive times,  then the first one is done(then notifyCheckpoint) ,  the offset of the fifth checkpoint will be commit. If failover and restart from first checkpoint, Wal-Log between [first_checkpoint, five_checkpoint) will be recycled.